.vale/fixtures/RedHat/GitLinks/.vale.ini

```diff
@@ -0,0 +1,5 @@
+; Vale configuration file to test the `GitLinks` rule
+StylesPath = ../../../styles
+MinAlertLevel = error
```

Comment:
Can we make this `MinAlertLevel = warning` Anyone that implements Vale in the CI will get failing builds if we merge this and they have errors. We had strong push back in the past about adding new Error rules.

Reply:
Will do.